storage/innobase/include/os0file.h

```diff
@@ -274,7 +274,9 @@ static const char ENCRYPTION_KEY_MAGIC_V2[] = "lCB";
 information version. */
 static const char ENCRYPTION_KEY_MAGIC_V3[] = "lCC";
 
-static const char ENCRYPTION_KEY_MAGIC_RK[] = "lRK";
+static const char ENCRYPTION_KEY_MAGIC_RK_V1[] = "lRK";
+
+static const char ENCRYPTION_KEY_MAGIC_RK_V2[] = "RKB";
```

Comment:
what are these? Can you comment what they stand for?

Reply:
done